packages/@react-types/shared/src/collections.d.ts

```diff
@@ -56,6 +56,7 @@ export interface SingleSelection {
 export type SelectionMode = 'none' | 'single' | 'multiple';
 export interface MultipleSelection {
   selectionMode?: SelectionMode,
+  disableEmptySelection?: boolean,
```

Comment:
any suggestion for naming this prop?

Reply:
Name sounds fine to me, but should it also be present in `SingleSelection`?

Reply:
added